### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by aevesdocker**
* add nested virtualization information

* enable nested virtualization

* tweaks

* tweaks

* review comments and additions

* review comments and additions

* review changes

* last checks

### CODE REVIEW

File: desktop/nested-virtualization.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+description: Instructions on how to enable nested virtualization
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
```
Comment (dockertopia) [inline desktop/nested-virtualization.md:8]: @aevesdocker just checking, is this the note you want published?

Comment (aevesdocker) [reply desktop/nested-virtualization.md:8]: Yes, thanks for checking. It's confirmed with the PM and engineering team

**Comment by aevesdocker** [comment]
@dockertopia yes, not adding to the TOC is done on purpose. We want to send the link to a few select clients before making it GA :)

**Review by dockertopia — APPROVED** [review]
LGTM

**Pull request merged**